cache-config/t3c-generate/t3c-generate.go

```diff
@@ -38,12 +37,12 @@ import (
 func main() {
 	cfg, err := config.GetCfg()
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Getting config: "+err.Error()+"\n")
+		log.Errorf("Getting config: %s\n", err.Error())
```

Comment:
This is a result of failing to initialize the config, so the loggers won't exist and this message will never be displayed.

So it needs to `fmt.Print`, just no way around it.

Reply:
Changed back to Fprintf